### PR TITLE
Vet-clinic: Create animal table

### PR DESCRIPTION
**Pull request by najibullahjafari**
Here:

- I used the recommended repo template.
- I created a database named vet_clinic.
- I create a table named animals with the following columns:
id: integer
name: string
date_of_birth: date
escape_attempts: integer
neutered: boolean
weight_kg: decimal
- I copied the SQL query in schema.sql
- I inserted the data:
- I wrote the recommended queries .
Here you can see the screenshots.
Screenshot 2023-08-30 073550
Screenshot 2023-08-30 073642
Screenshot 2023-08-30 074007
Screenshot 2023-08-30 081445
Screenshot 2023-08-30 081717
Screenshot 2023-08-30 081804
Screenshot 2023-08-30 082127
Screenshot 2023-08-30 082852

### CODE REVIEW

**Review by rloterh — COMMENTED** [review]
Hi Najibullah, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Kindly ensure that your ReadMe is professionally documented; Let's ensure that provided placeholder texts/ guide instructions are removed and/or replaced with appropriate contents applicable under each section of your ReadMe. 

**Kindly remove/update all aspects similar to the underlined:**

image

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: schema.sql [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+CREATE DATABASE vet_clinic;
+
+CREATE TABLE animals (
+    id INTEGER,
+    name VARCHAR(50),
+    date_of_birth DATE,
+    secape_attempts INTEGER,
+    neutered BOOLEAN,
+    weight_kg DECIMAL
+);
```
Comment (abdulhamiid) [inline schema.sql]: [OPTIONAL] - I will advise that you mark the `id` column as primary key

```sql
PRIMARY KEY (id)
```

A primary key is used to ensure data in the specific column is unique, it uniquely identifies each record in a table

👉 [SQL PRIMARY KEY Constraint](https://www.w3schools.com/sql/sql_primarykey.ASP)

File: schema.sql [hunk 2]
```diff
@@ -0,0 +1,11 @@
+/* Database schema to keep the structure of entire database. */
+CREATE DATABASE vet_clinic;
+
+CREATE TABLE animals (
+    id INTEGER,
+    name VARCHAR(50),
```
Comment (abdulhamiid) [inline schema.sql]: \[OPTIONAL\] I advise that you to make the `id` column not null and auto-incremented.

```sql
id int NOT NULL IDENTITY(1,1)
```

[AUTO INCREMENT Field](https://www.w3schools.com/sql/sql_autoincrement.asp)
Auto-increment allows a unique number to be generated automatically when a new record is inserted into a table.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,119 @@
...
+
+I would like to thank Microverse for their helpful practices.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (abdulhamiid) [inline README.md:116]: [REQUIRED] - Please fix the broken license link, let's avoid broken links in our readme for a professional repository 🤗🛠

**Review by abdulhamiid — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @najibullahjafari 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! :running_man: :runner: 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights 🌟

- ✔️ Descriptions are very descriptive
- ✔️ Create tables in SQL.
- ✔️ Insert and query data in SQL.

## Required Changes ♻️
_Please kindly check the comments under the review._

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by t-yanick — APPROVED** [review]
# STATUS: MILESTONE APPROVED :1st_place_medal: 

Hi @najibullahjafari , 

### Great job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Requested changes implemented 👍 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**